conf/messages.en

```diff
@@ -640,14 +633,25 @@ declaration.cusCode.error.empty = Enter a CUS code
 declaration.cusCode.error.length = CUS Code must be exactly 8 characters long
 declaration.cusCode.error.specialCharacters = CUS Code cannot contain special characters
 
-declaration.taricAdditionalCodes.header = Are there additional TARIC codes?
+declaration.taricAdditionalCodes.header = Is there an additional TARIC code?
 declaration.taricAdditionalCodes.header.hint = This will be the four digits that come after the commodity code in the {0}. If no additional codes are required, leave blank.
+declaration.taricAdditionalCodes.header.tradeTariff.link = UK Trade Tariff
 declaration.taricAdditionalCodes.error.empty = Enter a TARIC code
 declaration.taricAdditionalCodes.error.length = TARIC code must be exactly 4 characters long
 declaration.taricAdditionalCodes.error.specialCharacters = TARIC code cannot contain specialCharacters
 declaration.taricAdditionalCodes.add.hint = Add TARIC code
 declaration.taricAdditionalCodes.remove.hint = Remove TARIC code {0}
 
+declaration.nationalAdditionalCode.header = Do these goods require a national additional (NACT) code?
```

Comment:
But in the UI we're using `National additional code` so we can use this as variable. but it's okay